utilities_common/netstat.py

```diff
@@ -56,6 +56,42 @@ def ns_util(newstr, oldstr, delta, port_rate=PORT_RATE):
         util = rate/(port_rate*1000*1000*1000/8.0)*100
         return "{:.2f}%".format(util)
 
+def format_brate(rate):
+    """
+    Show the byte rate.
+    """
+    if rate == STATUS_NA:
+        return STATUS_NA
+    else:
+        rate = float(rate)
+        if rate > 1024*1024*10:
+            rate = "{:.2f}".format(rate/1024/1024.0)+' MB'
+        elif rate > 1024*10:
+            rate = "{:.2f}".format(rate/1024.0)+' KB'
+        else:
+            rate = "{:.2f}".format(rate)+' B'
+        return rate+'/s'
+
+def format_prate(rate):
+    """
+    Show the packet rate.
+    """
+    if rate == STATUS_NA:
+        return STATUS_NA
+    else:
+        #TODO what does it mean /s maybe p/s packets per seconds
+        return "{:.2f}".format(float(rate))+'/s'
+
+def format_util(brate, port_rate=PORT_RATE):
+    """
```

Comment:
Can we avoid assuming PORTE_RATE=40 here? I believe there are multitude of speeds and If speed in unavailable in config_db.json I am ok with displaying rate as N/A rather than using 40.

Reply:
already fixed